conf/uwsgi.ini

```diff
@@ -7,7 +7,7 @@ vacuum=True
 max-requests=5000
 env=DJANGO_SETTINGS_MODULE=laalaa.settings
 enable-threads=True
-processes=2
+processes=1
```

Comment:
Reducing to a single process for uwsgi seems a bit out of place here. Do we need this change?

Reply:
Don't need it per se I just thought it was the right thing to do when moving to duplicatable app pods, but if that's a misapprehension I'm happy to put it back

Reply:
I think without data to back the contention/usage, it's hard to tell. At this stage, shall we go with 2?